apps/mobile-wallet/src/components/layout/ScrollScreen.tsx

```diff
@@ -108,7 +108,6 @@ const ScrollScreen = ({
         alwaysBounceVertical={true}
         onScroll={handleScroll}
         onScrollEndDrag={scrollEndHandler}
-        style={{ overflow: SCREEN_OVERFLOW }}
```

Comment:
This comes from a cherry picked commit from one of the expo 52 PRs. This branch is not properly rebased on top of the other 6 branches. I wanted to fix something that I had already fixed.